### PR TITLE
feat: Add llm attribute to all transactions that contain llm spans for bedrock

### LINKED ISSUE #243: (bedrock): Add `llm` attribute to all transactions that contain LLM spans.
### Description
To help with classifying transaction data as AI generated, a new llm attribute of true will be added to all transactions that contain This helps with being able to separate the types of transactions within a single APM entity. 

### Acceptance criteria
 * When a transaction contains AWS bedrock LLM spans, add an attribute to the transaction of `llm: true`

### PR DESCRIPTION
**Pull request by svetlanabrennan**
## Description

Add llm: true attribute to Transactions that contain llm spans.

## How to Test

Added versioned tests

## Related Issues

Closes #243

### CODE REVIEW

File: lib/util.js [hunk 1]
```diff
@@ -5,6 +5,9 @@
 
 'use strict'
 const UNKNOWN = 'Unknown'
+const DESTINATIONS = {
```
Comment (jsumners-nr) [inline lib/util.js:8]: Is this this same thing as? https://github.com/newrelic/node-newrelic-aws-sdk/blob/e9584b4a43cbcc3d02b6663cac4c0e4d4d128741/lib/llm/event.js#L85-L90

Comment (svetlanabrennan) [reply lib/util.js:8]: Yes I believe so. 

This is how we're storing it in the node-newrelic repo: https://github.com/newrelic/node-newrelic/blob/main/lib/config/attribute-filter.js#L12-L15

Comment (jsumners-nr) [reply lib/util.js:8]: What I mean is, does your change need the full bitmask or only the 0x01 bit? If the full, then we should refactor my linked code to a central location and use it in both places.

Comment (svetlanabrennan) [reply lib/util.js:8]: It needs just the 0x01 for `TRANS_EVENT`.

**Review by jsumners-nr — APPROVED** [review]

**Pull request merged**